src/server/core/launch_server.c

```diff
@@ -21,7 +21,9 @@ static void listen_to_inputs(struct sockaddr_in *addr, int server_fd,
     data_t *data)
 {
     while (1) {
-        select_clients(addr, server_fd, data);
+        if (select_clients(addr, server_fd, data)) {
+            return;
```

Comment:
Je comprends pas la raison de return ici?

Reply:
`select_clients()` returns 0 if the server keeps running, 1 if the server must stop.